api/controllers/game/pass.js

```diff
@@ -56,7 +57,7 @@ module.exports = function (req, res) {
         };
         await Game.updateOne({ id: game.id }).set(gameUpdates);
         if (game.isRanked) {
-          await sails.helpers.addGameToMatch(game);
+          victory.currentMatch = await sails.helpers.addGameToMatch(game);
```

Comment:
We have a pattern of doing this elsewhere (read: above) but are we ok adding this directly to the victory object?